rover-config/50-rover-cameras.rules

```diff
@@ -6,11 +6,10 @@
 #
 # Install this in /etc/udev/rules.d
 
-# Mast camera
-SUBSYSTEM=="video4linux", ATTRS{idVendor}=="05a3", ATTRS{idProduct}=="9601", KERNELS=="1-2.3", SYMLINK+="video20"
+SUBSYSTEM=="video4linux", ATTRS{idVendor}=="05a3", ATTRS{idProduct}=="9230", KERNELS=="1-2.2", SYMLINK+="video20"
```

Comment:
Is this still the mast camera? Unless its not one single thing thing it would be nice to still have the comment above it for what it is.

Reply:
Whoops no it's still the mast cam, just deleted the comment by mistake